Makefile

```diff
@@ -54,16 +54,31 @@ all: binaries manpages
 help:
```

Comment:
Now the help is too long and unreadable

Reply:
I'll add `help-targets` target for added sub targets.

Reply:
added `help-targets` target and reverted `help` to simpler as previous.

Reply:
Main reason for "make help" was only to make "make menuconfig" happy

```

*** End of the configuration.
*** Execute 'make' to start the build or try 'make help'.
```